js/main.js

```diff
@@ -25,9 +25,86 @@ function getRandomFractionNumber(min, max, numberSymbols) {
 
   const randomNumber = min + Math.random() * (max - min);
 
-  return Number(randomNumber.toFixed(1 * numberSymbols));
+  return Number(randomNumber.toFixed(numberSymbols));
 }
 
-getRandomFractionNumber(1, 10, 2);
+getRandomFractionNumber(1, 10, 0);
 
 
+// Возвращает фотографию пользователя
+const AVATAR_COUNTS = 10;
+
+function getAvatarUrl () {
+  const randomNumber = getRandomFractionNumber(1, AVATAR_COUNTS, 0);
+
+  let AvatarUrl = 'img/avatars/userget' + `0${ randomNumber }.png`;
+  if (randomNumber === 10) {
+    AvatarUrl = 'img/avatars/userget' + `${ randomNumber }.png`;
+  }
+
+  return AvatarUrl;
+}
+
+const avatarArray = [];
+
+function getavatarValue () {
+  let nextValue;
+  while (avatarArray.length < AVATAR_COUNTS) {
+    nextValue = getAvatarUrl();
+    if(!avatarArray.includes(nextValue))
+    {avatarArray.push(nextValue);
+      return nextValue;
+    }
+  }
+}
+
+const createAvatarArray = Array.from({length: 10}, getavatarValue);
+
+
+// возвращает время заезда и выезда
+function getCheckinAndCheckout () {
+  const randomNumber = getRandomFractionNumber(12, 14, 0);
+  const checkinAndCheckout = `${ randomNumber }` + ':00';
```

Comment:
Опять редуцированная переменная, можно значени сразу вернуть без неё + не нужно складывать строки, линтер такое должен как ошибку подсвечивать, используй шаблоны ``
```suggestion
  return `${ randomNumber }:00`;
```